cmd/minikube/cmd/start.go

```diff
@@ -171,32 +173,43 @@ func init() {
 
 // initClusterUpFlags creates the CLI flags which needs to be passed on to 'oc cluster up'
 func initStartFlags() {
-	startFlagSet.String(isoURL, constants.DefaultIsoUrl, "Location of the minishift iso")
-	startFlagSet.String(vmDriver, constants.DefaultVMDriver, fmt.Sprintf("VM driver is one of: %v", constants.SupportedVMDrivers))
-	startFlagSet.Int(memory, constants.DefaultMemory, "Amount of RAM allocated to the minishift VM")
-	startFlagSet.Int(cpus, constants.DefaultCPUS, "Number of CPUs allocated to the minishift VM")
-	startFlagSet.String(humanReadableDiskSize, constants.DefaultDiskSize, "Disk size allocated to the minishift VM (format: <number>[<unit>], where unit = b, k, m or g)")
+	startFlagSet.String(isoURL, constants.DefaultIsoUrl, "Location of the minishift ISO.")
+	//NEEDINFO: Is this the same as the binary? we should be consistent in how we refer to it. Also, we need to change the default from jdyson.
+	startFlagSet.String(vmDriver, constants.DefaultVMDriver, fmt.Sprintf("The driver to use for the Minishift VM. Possible values: %v", constants.SupportedVMDrivers))
+	//NEEDINFO: If virtualbox is the virt provider how come kvm is the default driver and how are drivers related to this at all?
+	startFlagSet.Int(memory, constants.DefaultMemory, "Amount of RAM to allocate to the Minishift VM.")
+	startFlagSet.Int(cpus, constants.DefaultCPUS, "Number of CPU cores to allocate to the Minishift VM.")
+	startFlagSet.String(humanReadableDiskSize, constants.DefaultDiskSize, "Disk size to allocate to the Minishift VM. Use the format <size><unit>, where unit = b, k, m or g.")
 	startFlagSet.String(hostOnlyCIDR, "192.168.99.1/24", "The CIDR to be used for the minishift VM (only supported with Virtualbox driver)")
-	startFlagSet.StringSliceVar(&dockerEnv, "docker-env", nil, "Environment variables to pass to the Docker daemon. (format: key=value)")
+	//NEEDINFO: What is a CIDR?
+	startFlagSet.StringSliceVar(&dockerEnv, "docker-env", nil, "Environment variables to pass to the Docker daemon. Use the format <key>=<value>.")
+	//NEEDINFO: How do you separate multiple env vars? Comma? Space + comma? etc..
```

Comment:
This might be a good one for which we need to have a follow up issue. There are several things we need to test and verify yet, before we can change some of these things. Right now it does not have a high priority imo.